### PR TITLE
added memory usage calculation to replicated map when object type is …

### PR DESCRIPTION
**Pull request by bilalyasar**
…binary

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalReplicatedMapStatsImpl.java [hunk 1]
```diff
@@ -113,11 +116,12 @@ public void setBackupCount(int backupCount) {
 
     @Override
     public long getOwnedEntryMemoryCost() {
-        return 0;
+        return ownedEntryMemoryCost;
     }
 
     // TODO: unused
```
Comment (mesutcelik) [inline hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalReplicatedMapStatsImpl.java]: you can remove `TODO`.

**Comment by mesutcelik** [comment]
:+1:  one minor comment

**Comment by bilalyasar** [comment]
@emrahkocaman can you review this PR please?

**Comment by emrahkocaman** [comment]
Tests are needed to be updated.

image

**Comment by emrahkocaman** [comment]
:+1: when pr builder passes

**Pull request merged**